streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -67,43 +76,45 @@
 import static org.apache.kafka.common.utils.Utils.getPort;
 
 /**
- * Kafka Streams allows for performing continuous computation on input coming from one or more input topics and
- * sends output to zero or more output topics.
+ * {@code KafkaStreams} allows for performing continuous computation on input coming from one or more input topics and
```

Comment:
Matter of taste I guess. Can update it though. (btw: `{@code XXX}` is the same as `<code>XXX</code>` -- there is no link; it's just formatting markup.